frontend/src/actionCreators/varianceActionCreator.js

```diff
@@ -54,14 +52,26 @@ export const fetchVariancesByMine = ({ mineGuid }) => (dispatch) => {
     .then((response) => {
       dispatch(success(reducerTypes.GET_MINE_VARIANCES));
       dispatch(varianceActions.storeVariances(response.data));
-      dispatch(hideLoading());
     })
     .catch(() => dispatch(error(reducerTypes.GET_MINE_VARIANCES)))
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchVariancesById = (mineGuid, varianceGuid) => (dispatch) => {
```

Comment:
`fetchVarianceById`? single? or does it do something else.

Reply:
Correct, it is fetching a single variance.

aaand good catch! will remove the `s`